create-packages.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
```

Comment:
I think you don't need create such a file. You can rule commonjs and ESM imports in your package.json file. Example: https://github.com/codex-team/config-loader/blob/build-lib/package.json#L16-L21

Reply:
It is not the same, read the description of the PR

Reply:
I'm using the Conditional export in main package.json file for defining paths to the bundles. But this file is creating package.json files for defining of **package types** of different bundles.

Reply:
I don't believe that we need to create separate package.json files for ESM and CJS. I want to check it on my own.

I thought that projects that do not support ESM will always use CJS regardless of what is specified in `type` field.